docs/source/user-guide/configs.md

```diff
@@ -49,7 +49,7 @@ Environment variables are read during `SessionConfig` initialisation so they mus
 | datafusion.execution.target_partitions                     | 0          | Number of partitions for query execution. Increasing partitions can increase concurrency. Defaults to the number of CPU cores on the system                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | datafusion.execution.time_zone                             | +00:00     | The default time zone Some functions, e.g. `EXTRACT(HOUR from SOME_TIME)`, shift the underlying datetime according to this time zone, and then extract the hour                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-| datafusion.execution.parquet.enable_page_index             | false      | If true, uses parquet data page level metadata (Page Index) statistics to reduce the number of rows decoded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| datafusion.execution.parquet.enable_page_index             | true       | If true, uses parquet data page level metadata (Page Index) statistics to reduce the number of rows decoded.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
I think here we also skip IO when the whole page is skipped.

Reply:
I will refine the description as a follow on PR

Reply:
https://github.com/apache/arrow-datafusion/pull/6342